src/httprpc.cpp

```diff
@@ -184,6 +191,23 @@ static bool HTTPReq_JSONRPC(HTTPRequest* req, const std::string &)
         // Set the URI
         jreq.URI = req->GetURI();
 
+#ifdef ENABLE_WALLET
+        if (walletName.empty()) {
```

Comment:
That would be less clean when calling in from GUI, tests, etc. But maybe a `void*` would make sense on the class?